go.mod

```diff
@@ -174,17 +176,17 @@ require (
 	github.com/rogpeppe/go-internal v1.8.1 // indirect
 	go.uber.org/atomic v1.11.0 // indirect
 	k8s.io/apiextensions-apiserver v0.27.2
-	knative.dev/reconciler-test v0.0.0-20230503093155-937cf5922570
+	knative.dev/reconciler-test v0.0.0-20230803013451-a8041ba726ed
```

Comment:
This is pointing to main, commit `https://github.com/knative-extensions/reconciler-test/commit/a8041ba726ed`

Reply:
👍 
I hope [this](https://github.com/openshift-knative/serverless-operator/pull/2207/commits/043ca97c29b7dc9acea6a952446dfdeab35be5f5#diff-5b45c8cc2cb795405146417d3efa55bc2472f989e32bef288f7493e190f1388bL101) is compatible?

Reply:
https://github.com/knative-extensions/reconciler-test/pull/566